cdk/lib/newswires.ts

```diff
@@ -71,9 +78,15 @@ export class Newswires extends GuStack {
 					DATABASE_PORT: database.dbInstanceEndpointPort,
 					DATABASE_NAME: databaseName,
 				},
+				vpc,
+				vpcSubnets: {
+					subnets: privateSubnets,
+				},
 			},
 		);
 
+		ingestionLambda.connections.allowTo(database, Port.tcp(5432));
```

Comment:
would the following not work?
```suggestion
	database.grantConnect(ingestionLambda);
```
(taken from an RDS Proxy example, so might legit not work, but if it does I think more readable)

Reply:
documentation here https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-rds.DatabaseInstance.html#grantwbrconnectgrantee implies it doesn't work though lol, although linked issue suggests it now does work https://github.com/aws/aws-cdk/issues/11851#issuecomment-1531639694

Reply:
we have this in https://github.com/guardian/newswires/blob/71041b3923888dc23cdbd73e7a9604cae2aeaef0/cdk/lib/newswires.ts#L107 - `grantConnect` grants the IAM permission, but we also need to grant network access through security groups, which is what `connections.allowTo` grants